utilities/wallet-tester/src/common/helpers/buildUnsignedTx.ts

```diff
@@ -221,7 +218,15 @@ export default async function buildUnsignedTx(
   }
 
   // build a full transaction, passing in empty witness set
-  const unsignedTx = Transaction.new(txBuilder.build(), TransactionWitnessSet.new(), auxMetadata);
+  const txBody = txBuilder.build();
+  
+  // #15 add network id
+  if (builder.networkId && [0, 1].includes(Number(builder.networkId))) {
+    const networkId = Number(builder.networkId) === 0 ? NetworkId.testnet() : NetworkId.mainnet()
+    txBody.set_network_id(networkId);
```

Comment:
I've tested this fix and when I included the network ID in the form then it was correctly set on the trasaction body. However the fee calculated for the transaction didn't get higher compared to a transaction that doesn't include the network_id. I think calling `txBody.set_network_id(networkId);` after `const txBody = txBuilder.build();` causes the fee to not include the network_id bytes, thus the calculated fee is not enough to cover for the transaction.